src/main/java/uk/sky/cqlmigrate/CqlMigratorImpl.java

```diff
@@ -105,18 +102,16 @@ public void clean(String[] hosts, int port, String keyspace) {
      * {@inheritDoc}
      */
     public void clean(Session session, String keyspace) {
-        session.execute("DROP KEYSPACE IF EXISTS " + keyspace);
+        Statement clean = new SimpleStatement("DROP KEYSPACE IF EXISTS " + keyspace)
+                .setConsistencyLevel(cqlMigratorConfig.getWriteConsistencyLevel());
```

Comment:
I can't find documentation to define whether or not it is.  Can @chbatey comment on this - you guys were reviewing together, no?